src/main/java/Pet.java

```diff
@@ -1,6 +1,93 @@
+import org.testng.annotations.Test;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
 
 public class Pet {
+String baseurl="https://petstore.swagger.io/v2/";
```

Comment:
1) Pls format your code, I see style format issues
2) Use RestAssured.baseURI for Setting BaseURI at the global level, so you don't have to pass this string value into the tests